crates/uv-python/src/downloads.rs

```diff
@@ -66,6 +66,8 @@ pub enum Error {
     },
     #[error("Invalid download URL")]
     InvalidUrl(#[from] url::ParseError),
+    #[error("Invalid download URL: {0}")]
+    InvalidUrlFormat(Url),
```

Comment:
I'm not sure what caused this in #13376, a data URL maybe? The url crate docs say (`cannot_be_a_base`):

> This is the case if the scheme and `:` delimiter are not followed by a `/` slash,
> as is typically the case of `data:` and `mailto:` URLs.